.github/CODEOWNERS

```diff
@@ -35,6 +35,7 @@
 /corehq/apps/commtrack/ @esoergel
 /corehq/apps/commtrack/static/ @esoergel @orangejenny
 /corehq/apps/commtrack/templates/ @esoergel @orangejenny
+/corehq/apps/data_dictionary/ @esoergel
```

Comment:
```suggestion
/corehq/apps/data_dictionary/ @esoergel @orangejenny 
```

Reply:
(went to add myself to the UI files, then realized from a product perspective i'm also interested in server-side DD changes)